test/jdk/java/lang/constant/MethodTypeDescTest.java

```diff
@@ -62,14 +61,14 @@ private void testMethodTypeDesc(MethodTypeDesc r) throws ReflectiveOperationExce
         assertEquals(r, MethodTypeDesc.of(r.returnType(), r.parameterList().toArray(new ClassDesc[0])));
         assertEquals(r, MethodTypeDesc.of(r.returnType(), r.parameterList().stream().toArray(ClassDesc[]::new)));
         assertEquals(r, MethodTypeDesc.of(r.returnType(), IntStream.range(0, r.parameterCount())
-                                                                   .mapToObj(r::parameterType)
-                                                                   .toArray(ClassDesc[]::new)));
+                .mapToObj(r::parameterType)
```

Comment:
this reformatting may be accidental.  Please keep the original format.   Same applies to other reformatted lines in this file.